packages/components/src/duotone-picker/utils.js

```diff
@@ -26,7 +29,7 @@ export function getDefaultColors( palette ) {
 	return palette
 		.map( ( { color } ) => ( {
 			color,
-			brightness: tinycolor( color ).getBrightness() / 255,
+			brightness: colord( color ).brightness(),
```

Comment:
I anyone else is wondering about `/ 255` removal: `colord().brightness()` returns number from 0-1, while `tinycolor().getBrightness` from 0-255.

Reply:
Exactly colord brightness already returns a value from 0 to 1 as we want so there is no need to divide by 255.